src/metrics.cpp

```diff
@@ -517,9 +518,9 @@ void ThreadShowMetricsScreen()
 
         // Thank you text
         std::cout << _("Zen is economic freedom. Thanks for running a node.") << std::endl;
-        std::cout << _("仕方が無い") << std::endl;
-        std::cout << _("Shikata ga nai.") << std::endl;
-        std::cout << _("它不能得到帮助") << std::endl << std::endl;
+
+        std::cout << "Software Version: " << FormatVersion(CLIENT_VERSION);
+        std::cout << std::endl;
```

Comment:
Can we now merge all together on one line?

```suggestion
        std::cout << "Software Version: " << FormatVersion(CLIENT_VERSION) << std::endl;
```

Reply:
done